projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.model.ts

```diff
@@ -0,0 +1,7 @@
+import { Observable } from 'rxjs';
```

Comment:
I believe this file exist already in the ./model and will need to be updated accordingly. 

Tbh, you can leave this here and remove the ./model one in *merchandising-carousel* dir. Either way works fine, but just update the paths in other folders

Reply:
Deleted this one as the model was the most widely used one